library/src/main/java/com/getstream/sdk/chat/StreamChat.java

```diff
@@ -39,6 +33,7 @@
     private static Integer lastUnreadChannels;
     private static boolean lifecycleStopped;
     private static boolean userWasInitialized;
+    private static Context mContext;
```

Comment:
Let's drop `m` prefix. We're not using it elsewhere, so no reason to have it here.